src/manifest.json

```diff
@@ -23,7 +23,7 @@
     },
     "devtools_page": "Devtools/devtools.html",
     "web_accessible_resources": ["insights.html", "assessments/*", "injected/*", "background/*", "common/*", "DetailsView/*", "bundle/*"],
-    "permissions": ["storage", "webNavigation", "tabs", "notifications", "activeTab"],
+    "permissions": ["storage", "webNavigation", "tabs", "notifications", "https://*/*", "http://*/*", "file://*/*"],
```

Comment:
I would lean towards keeping activeTab in addition to the origin permissions because I suspect it might reduce a "this extension requires new permissions" prompt when we switch back later

Reply:
Forgot we don't already have activeTab in production, so we have the same risk of the prompt either way, nevermind this